server/src/internalClusterTest/java/org/elasticsearch/snapshots/SharedClusterSnapshotRestoreIT.java

```diff
@@ -2072,6 +2072,57 @@ public void testThrottling() throws Exception {
             .putNull(INDICES_RECOVERY_MAX_BYTES_PER_SEC_SETTING.getKey()).build()).get();
     }
 
+    public void testDynamicRestoreThrottling() throws Exception {
```

Comment:
I think this test would have passed before this change, although maybe a bit slower. Doesn't seem to be much slower, however. Can we somehow assert that it really does run at full speed at the end?

(I couldn't think of a simple way to do this, we can leave it as-is if you can't either)

Reply:
I've pushed a5c576b. Let me know what you think